src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,52 @@
+package racingcar;
+
+import static racingcar.CarConstant.*;
+
+public class Car {
+    private final String carName;
+    private String moveString;
+    private int position = INIT_POSITION;
+
+    public Car(String carName) {
+        validateCarName(carName);
+        this.carName = carName;
+    }
+
+    private void validateCarName(String carName) {
+        if (carName.length() > MAX_NAME_LENGTH) {
```

Comment:
```suggestion
        if (carName.length() >= MAX_NAME_LENGTH) {
```
기능요구사항에 자동차이름은 5자까지 가능합니다.